python/ray/tune/tune.py

```diff
@@ -299,6 +299,10 @@ def run(
         TuneError: Any trials failed and `raise_on_failed_trial` is True.
     """
 
+    # at this point, locals is equal to args and kwargs
```

Comment:
Can you beef it up a bit so that people don't accidentally add stuff prior to this line.